plugins.json

```diff
@@ -647,6 +647,18 @@
         "remote_install": true,
         "compatible_version": ">=1.0.37"
     },
+    "spectre": {
+        "title": "Spectre Theme",
+        "version": "1.0.2",
+        "author": "Viggo",
+        "license": "CC-BY-4.0",
+        "description": "Modern And Simple CSS Theme For Kanboard",
+        "homepage": "https://github.com/kuerme/kanboard-theme-spectre",
+        "readme": "https://raw.githubusercontent.com/kuerme/kanboard-theme-spectre/master/README.md",
+        "download": "https://github.com/kuerme/kanboard-theme-spectre/releases/download/v1.0.2/Spectre.zip",
```

Comment:
The download URL returns a 404.

It should be `https://github.com/kuerme/kanboard-theme-spectre/releases/download/v1.0.2/Spectre-1.0.2.zip`